v03_pipeline/lib/reference_data/clinvar_test.py

```diff
@@ -191,6 +197,12 @@ def test_join_to_submission_summary_ht(self, mock_download):
                         'PreventionGenetics, part of Exact Sciences',
                         'Invitae',
                     ],
+                    clinical_significances=[
```

Comment:
this looks like something that we can maybe enum map?  

the process for that would look like:
- using hail to aggregate to a set and inspecting the values.
- adding the list submitters to the `enum_select` field nested under `clinvar` in `config.py`.